.setup/update_system.sh

```diff
@@ -23,9 +23,28 @@ if [ -d "${CURRENT_DIR}/../.vagrant" ]; then
     VAGRANT=1
 fi
 
+APT=true
+if [ ${CI} == true ]; then
+    APT=false
+fi
```

Comment:
Why disable this on CI in particular?

Reply:
e2e tests were failing and I couldn't figure out the issue so tried disabling it on CI as advised but that didn't work, using bash instead of source did, I've reverted this.